src/index.js

```diff
@@ -1,53 +1,43 @@
-import React from 'react'
+import React, {Component} from 'react'
 import PropTypes from 'prop-types'
-import createReactClass from 'create-react-class'
-import { createFilter } from './util'
+import {createFilter} from './util'
 
-const Search = createReactClass({
-  propTypes: {
-    className: PropTypes.string,
-    inputClassName: PropTypes.string,
-    onChange: PropTypes.func,
-    caseSensitive: PropTypes.bool,
-    sortResults: PropTypes.bool,
-    fuzzy: PropTypes.bool,
-    throttle: PropTypes.number,
-    filterKeys: PropTypes.oneOf([
-      PropTypes.string,
-      PropTypes.arrayOf(PropTypes.string)
-    ]),
-    value: PropTypes.string
-  },
-
-  getDefaultProps () {
-    return {
-      className: '',
-      onChange () {},
-      caseSensitive: false,
-      fuzzy: false,
-      throttle: 200
-    }
-  },
-
-  getInitialState () {
-    return {
+class Search extends Component {
+  constructor (props) {
+    super(props)
+    this.state = {
       searchTerm: this.props.value || ''
     }
-  },
+    this.updateSearch = this.updateSearch.bind(this)
```

Comment:
we need to bind `filter` as well, right?

Reply:
Yes, you're right. Good catch. I made a build and some tests and got no problem. This is weird.

Fixed.